use em instead of pt in default.css

xiota opened the pull request:
Using rem instead of pt in the css allows the output to scale with font size.  I also ran the stylesheet through a beautifier, added Courier Prime to the font list, and adjusted the margins of #wrapper.  Before/after screenshots are attached to show the effect of the margin adjustment.

* [screenshot-css-before](https://user-images.githubusercontent.com/8353098/133702618-f05e238c-4a56-4f23-b1cb-e68eb0847b51.png)
* [screenshot-css-after](https://user-images.githubusercontent.com/8353098/133702625-d2a7376d-75c6-4e7d-92e4-6ca919a40d85.png)

Here's a screenshot showing a screenplay preview scaled down: [screenshot-scaled-down](https://user-images.githubusercontent.com/8353098/133706575-e383ded0-6a23-4acd-b624-37363848dd55.png)

vilcans commented:
That's a lot of changes in one commit.

Courier Prime should be in the font list, that's good.

I'm no expert on the details of em vs rem, but as far as I understand the difference is that em is relative to the size of the element the style is for, while rem is relative to the HTML body. Shouldn't scaling work just as well with both? Also some of the rem values are different from the original em values, why is that?

The new margins - why that change? We have to be careful with those as there's sort of a standard for how large margins a screenplay should have. I think Jonathan Poritsky who made the CSS probably thought about that.

xiota commented:
rem vs em... Since screenplays mostly use the same font size throughout, there isn't much difference.  Maybe em would be better in this case so that #wrapper can be scaled independently if embedded in another document.  I also just looked up the user-select tag, and it seems [modern browsers](https://caniuse.com/user-select-none) support it properly, so in addition to changing rem to em, I've left only the standard user-select.

HTML is not a good platform for exact placement.  Browsers commonly resize pages to fit the printer and cut off text and images between pages.  For printing, the pdf output of screenplain would be much better.

The original values in the stylesheet didn't add up to the width of either A4 or letter paper (I changed it to correspond to A4).  When the border isn't shown, it looks okay, but the left and right margins were badly skewed, and the bottom margin was 0.  It would look bad embedded in another document with the border shown.  There were also some nonsensical values.  Like negative margins, and a -1pt margin.  (I don't think screenplain uses those tags.)  It seems like it may have been designed for a now-discontinued browser.

xiota commented:
I recalculated the margins from measurements at [Nashville Film Institute](https://www.nfi.edu/screenplay-format/) and [Scriptwriting Secrets](http://www.scriptwritingsecrets.com/DualDialogue.htm).  Measurements from other sites were more or less consistent with those two.  Some gave a range or figures that differed by a couple characters.

I removed the browser reset and just included the necessary changes in the tags themselves.  This should allow embedding screenplays on other pages that aren't guaranteed to have a reset css.

I removed the styling that isn't used by screenplain.  That takes care of the tags with unusual values.

The margins on #wrap *should* print "properly" with 0.5in left/right and 1.0in top/bottom margins added in the print dialog. But browsers break at odd places, and have other quirks.  I added avoid break instructions, but I don't know of any browser that actually follows them.

The result is slightly different from screenplain pdfs.  The pdf uses 61char, but the measurements most recommended online result in 60char line lengths.  Some example screenplays online have 58 char line lengths.
